Havit.Blazor.Components.Web.Bootstrap/Forms/SearchBox/HxSearchBox.nongeneric.cs

```diff
@@ -19,6 +19,7 @@ static HxSearchBox()
 			ClearIcon = BootstrapIcon.XLg,
 			MinimumLength = 2,
 			Delay = 300,
+			HighlightFirstSuggestion = true
```

Comment:
Did we change the default intentionally?
Do we want this to be a new default + documented breaking change (also for HxAutosuggest, maybe for HxInputTags).
Do we want to remove the parameter completely and make this a standard behavior?
cc @crdo

Reply:
removed